views/partials/banner_breach.hbs

```diff
@@ -13,6 +13,6 @@
     <span class="breach-info"><span class="medium">Compromised Data</span> {{breachDataClasses breach.meta.DataClasses }}</span>
     <span class="breach-info"><span class="medium">Compromised Accounts</span> {{localeString breach.meta.PwnCount }}</span>
     <span class="breach-info">{{{ breach.meta.Description }}}</span>
-    <span class="source-info">Breach data provided by<span class="demi"> Have I Been Pwned?</span></span>
```

Comment:
Final nonsense random thought... Should we add a `target="_blank" rel="noopener noreferrer"` to open this in a new tab (so we don't lose our current page/results)?